migration/src/m20250526_070842_idx_asset_supply_burnt_owner_type.rs

```diff
@@ -0,0 +1,37 @@
+use sea_orm::{ConnectionTrait, DatabaseBackend, Statement};
```

Comment:
@Nagaprasadvr is this strictly needed? If not please add in a separate PR so can release the changes to the query and then see about adding indexes.